package.json

```diff
@@ -148,7 +148,7 @@
     "@oclif/plugin-command-snapshot": "^2.0.0",
     "@salesforce/dev-config": "^2.1.2",
     "@salesforce/dev-scripts": "^0.9.11",
-    "@salesforce/plugin-release-management": "^2.2.4",
+    "@salesforce/plugin-release-management": "dev",
```

Comment:
This will be updated to the correct version once the `plugin-release-management` PR is merged.

Reply:
that merged, please update